### PR TITLE
Card details

### PR DESCRIPTION
**Pull request by WE-Kaito**
pls check :)

image

### CODE REVIEW

File: frontend/src/hooks/useStore.ts [hunk 1]
```diff
@@ -92,4 +88,14 @@ export const useStore = create<State>((set) => ({
             .then(() => set({isLoading: false}));
 
     },
+
+    selectCard: (card: CardType) => {
+        set({selectedCard: card});
+        console.log(card);
```
Comment (toshydev) [inline frontend/src/hooks/useStore.ts:94]: Forgot to remove or do you need it?
Nevermind. Just wanted to leave a comment in your code xD
```suggestion

```

File: frontend/src/pages/Deckbuilder.tsx [hunk 2]
```diff
@@ -1,19 +1,34 @@
...
+import CardDetails from "../components/CardDetails.tsx";
 
 export default function Deckbuilder() {
 
     const fetchCards = useStore((state) => state.fetchCards);
+    const selectedCard = useStore((state) => state.selectedCard);
+    const hoverCard = useStore((state) => state.hoverCard);
 
     useEffect(() => {
+        // @ts-ignore
```
Comment (toshydev) [inline frontend/src/pages/Deckbuilder.tsx:16]: What's `@ts-ignore`?

**Review by toshydev — APPROVED** [review]
This is next level bre! 🤯 Can't wait to play this!

File: frontend/src/hooks/useStore.ts (hunk 1 above)
Comment (WE-Kaito) [reply frontend/src/hooks/useStore.ts:94]: this is just so i can control the card values when clicking. Was pretty helpful so far, will remove it in the future :D

File: frontend/src/pages/Deckbuilder.tsx (hunk 2 above)
Comment (WE-Kaito) [reply frontend/src/pages/Deckbuilder.tsx:16]: to supress the typescript error. The function has default values but somehow it's marked as an error

**Pull request merged**